test-worker/test/asset-compute/worker/validate-image.sh

```diff
@@ -35,7 +35,7 @@ function im_identify() {
 
 # check for pixel equality with some slight accepted color value difference
 # will exit with 1 if images differ
-FUZZ="5%"
+FUZZ="10%"
```

Comment:
why increase FUZZ ? AFAICS, This isn't used in colorspace validation

Reply:
Fuzz got increase to not flag tests as failed when renditions got created by 2 different imagemagick versions (note that in other tests it's up to 15%). And the fuzz value is used - the "normal" validation script (this one) gets called after any specific/custom validation.

Reply:
Ok thanks!
I know one imagemagick is the one used inside https://github.com/adobe/asset-compute-sdk/blob/master/test-worker/manifest.yml#L7 , which is the other one(for this scenario)?

Reply:
The one from our latest base image, 7.0.11-6 (with latest supporting libraries too).